x-pack/metricbeat/module/sql/query/_meta/data.json

```diff
@@ -10,372 +10,495 @@
-                "qcache_free_memory": 1031832,
-                "qcache_hits": 0,
-                "qcache_inserts": 0,
-                "qcache_lowmem_prunes": 0,
-                "qcache_not_cached": 4,
-                "qcache_queries_in_cache": 0,
-                "qcache_total_blocks": 1,
-                "queries": 18,
-                "questions": 1,
-                "select_full_join": 0,
-                "select_full_range_join": 0,
-                "select_range": 0,
-                "select_range_check": 0,
-                "select_scan": 0,
-                "slave_open_temp_tables": 0,
-                "slow_launch_threads": 0,
-                "slow_queries": 0,
-                "sort_merge_passes": 0,
-                "sort_range": 0,
-                "sort_rows": 0,
-                "sort_scan": 0,
-                "ssl_accept_renegotiates": 0,
-                "ssl_accepts": 0,
-                "ssl_callback_cache_hits": 0,
-                "ssl_client_connects": 0,
-                "ssl_connect_renegotiates": 0,
-                "ssl_ctx_verify_depth": 0,
-                "ssl_ctx_verify_mode": 0,
-                "ssl_default_timeout": 0,
-                "ssl_finished_accepts": 0,
-                "ssl_finished_connects": 0,
-                "ssl_session_cache_hits": 0,
-                "ssl_session_cache_misses": 0,
-                "ssl_session_cache_overflows": 0,
-                "ssl_session_cache_size": 0,
-                "ssl_session_cache_timeouts": 0,
-                "ssl_sessions_reused": 0,
-                "ssl_used_session_cache_entries": 0,
-                "ssl_verify_depth": 0,
-                "ssl_verify_mode": 0,
-                "table_locks_immediate": 118,
-                "table_locks_waited": 0,
-                "table_open_cache_hits": 0,
-                "table_open_cache_misses": 0,
-                "table_open_cache_overflows": 0,
-                "tc_log_max_pages_used": 0,
-                "tc_log_page_size": 0,
-                "tc_log_page_waits": 0,
-                "threads_cached": 0,
-                "threads_connected": 1,
-                "threads_created": 1,
-                "threads_running": 1,
-                "uptime": 4,
-                "uptime_since_flush_status": 4
-            },
-            "string": {
-                "compression": "OFF",
-                "innodb_buffer_pool_dump_status": "Dumping of buffer pool not started",
-                "innodb_buffer_pool_load_status": "Buffer pool(s) load completed at 200126 13:05:12",
-                "innodb_buffer_pool_resize_status": "",
-                "max_used_connections_time": "2020-01-26 13:05:13",
-                "ssl_cipher": "",
-                "ssl_cipher_list": "",
-                "ssl_server_not_after": "",
-                "ssl_server_not_before": "",
-                "ssl_session_cache_mode": "NONE",
-                "ssl_version": ""
-            }
+            "aborted_clients": 0,
```

Comment:
This file is gonna be embedded into some places of documentation. Can we generate a smaller example, please? :slightly_smiling_face:

Reply:
Yes, this file was large currently. https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-sql-query.html
Updated to have a smaller data.json file so that the external doc is better readable.